src/SFA.DAS.AssessorService.Database/PostDeploymentScripts/EPAO_Migration_Part2_OrganisationStandard.sql

```diff
@@ -0,0 +1,15 @@
+﻿-- migration part 2 OrganisationStandard
+
+MERGE INTO OrganisationStandard MasterOS
+USING (
+SELECT os1.Id, [IfateReferenceNumber] from OrganisationStandard os1
+LEFT JOIN (select [IfateReferenceNumber] , [Larscode] FROM standards WHERE LarsCode != 0 GROUP BY [IfateReferenceNumber] , [Larscode]
+UNION SELECT 'ST0107',52 
```

Comment:
How come we are selecting on two specific IFateReferenceNumbers? I don't think these are special cases.

Reply:
They are. They are no longer even on IFATE web site or LARS. Conveniently brushed uner the carpet as they did not fit the new data. They are historical. but we have some older RoEPAO entries for those LARS codes, and these need to be accessible via the IFATE Reference Number.